### DISCUSSION TITLE
flip_constrained failed !

### DISCUSSION THREAD

**Discussion by vbakis** [phoebe-project/phoebe2#801] [Implementation]
Hello,

I received the following error while implementing the flip_constrained in Workshop Tutorial_12 . Any idea?

Thanks,

VB

![flip_error](https://github.com/phoebe-project/phoebe2/assets/80822881/0297b8e9-c964-499b-828e-983de174ddf0)

**Comment by kecnry** [comment]
Can you let us know which version of PHOEBE you have installed and which workshop this is from (you can just link to the tutorial directly)?  Have you made any other modifications to the notebook or is it failing to run as-is?  The workshop tutorials are meant to run on a specific version of PHOEBE at the time of the workshop and are not kept up to date with any changes (unlike the official tutorials which _should_ be up-to-date).
